utils/automation/fossologyscanner.py

```diff
@@ -362,16 +362,18 @@ def main(parsed_args):
                         parser.php_components != [] or
                         parser.npm_components != [] )
 
-    python_comps = parser.python_components
-    unsupported_comps = parser.unsupported_components
-
-    if len(python_comps) != 0:
+    if len(parser.python_components) != 0:
       python_parser = PythonParser()
-      python_list = python_parser.parse_components(python_comps)
+      python_list = python_parser.parse_components(parser.python_components)
       download_list += python_list
 
-    if len(unsupported_comps) != 0:
-      for comp in unsupported_comps:
+    if len(parser.npm_components) != 0:
```

Comment:
python takes empty lists as `false`
`if parser.python_components:` will also do the same job.